scaladoc/src/dotty/tools/scaladoc/tasty/SyntheticSupport.scala

```diff
@@ -114,4 +114,3 @@ trait SyntheticsSupport:
     typeForClass(c).asInstanceOf[dotc.core.Types.Type]
       .memberInfo(symbol.asInstanceOf[dotc.core.Symbols.Symbol])
       .asInstanceOf[TypeRepr]
-
```

Comment:
only empty line removal?

Reply:
Probably some leftover after debugging

Reply:
Oh, it's probably autoformatter of VSCode that trim trailing newlines at the end of the file